pkgs/data/themes/gruvbox-gtk-theme/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gnome-themes-extra
+, gtk-engine-murrine
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "gruvbox-gtk-theme";
+  version = "unstable-2022-12-09";
+
+  src = fetchFromGitHub {
+    owner = "Fausto-Korpsvart";
+    repo = pname;
```

Comment:
Use the exact same name here used in the github project url.

Reply:
Ah yes, I thought that was the case, but I was recommended to switch to pname.